08-transport.Rmd

```diff
@@ -510,9 +569,14 @@ igraph::V(g)$m = m$membership
 gdf = as_long_data_frame(g)
 ```
 
-
 One can also find the shortest route between origins and destinations using this graph representation of the route network:
 
+<!-- 
+1. figure caption is missing 
+2. not really clear what you are doing here, what is sum_network_routes doing, why 1 and 20, what is length referring to?
+3. Since this is the local routing we should back-reference remote routing
+4. Why is there a section routes and a separate section route networks?
+-->
 ```{r}
```

Comment:
1. figure caption is missing 
2. not really clear what you are doing here, what is sum_network_routes doing, why 1 and 20, what is length referring to?
3. Since this is the local routing we should back-reference remote routing
4. Why is there a section routes and a separate section route networks?

Reply:
In reply @jannes-m - these are good comments worthy of responses:

1. No longer plotted - was a plot-heavy chapter and can be a reward encouraging readers to run the code.
2. I've explained what's going on and that 1 and 20 are arbitrary (sure a better example could be found but this is good to demonstrate local routing I think) and explained how to select nodes by geographic location.
3. Done
4. To coincide with the principle of structuring our book, and the chapters within it, based on difficulty and frequency with which the methods will be needed (routes are easier and more commonly needed than route networks)

Sound reasonable? Seem fixed to you?

Reply:
Yes, perfect! Only one remark with regard to 4. This is a good argument. Maybe we should state so in the text or at the beginning of route networks, that they are more complex and therefore presented in a separate section